source/numerics.tex

```diff
@@ -6752,12 +6752,6 @@
 matrix class nor a field class.
 However, it is a very useful building block for designing such classes.}
 
-\pnum
-An implementation is permitted to qualify any of the functions declared in
-\tcode{<valarray>}
-as
-\tcode{inline}.
-
```

Comment:
Hmm, this is erasing the whole paragraph rather than just the tcode bit. Was that intended?